### PR TITLE
Added test for creation date workitems

### PR DESCRIPTION
**Pull request by naina-verma**
Added tests for time displayed on detail page 
- Added a function which create mock time
- Automated tests for mobile/ desktop to check date is displaying as expected

### CODE REVIEW

File: src/tests/work-item/work-item-list/work-item-timeStamp.spec.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    page.workItemByURLId("id4");
+    expect(detailPage.getCreatedtime()).toBe('15 days ago');
+    page.workItemByURLId("id5");
+    expect(detailPage.getCreatedtime()).toBe('25 days ago');
+    page.workItemByURLId("id6");
+    expect(detailPage.getCreatedtime()).toBe('a month ago');
+    page.workItemByURLId("id7");
+    expect(detailPage.getCreatedtime()).not.toBe('a year ago');
+   });
+ it('check date showing up correctly - phone', function () {
```
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/work-item-timeStamp.spec.js]: Yes - we should focus on desktop mode, but it's easy to also add tests for the phone screens.

Comment (naina-verma) [inline src/tests/work-item/work-item-list/work-item-timeStamp.spec.js]: @debloper , i have done the requested changes by you. 
Thanks :)

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 2]
```diff
@@ -38,7 +38,9 @@ class WorkItemListPage {
      browser.get("http://localhost:8088/");
    }
  };
-
+ workItemByURLId (workItemId) {
```
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: I like the addition of the function that enables us to access a workitem's detail page directly through the detailpage URL! In fact, I'd like to see us add many more tests that access the UI's element through their respective direct URLs. 

I did find a way to resolve the problem that you are having. The key is to first access the workitem's detail page, and then access the date on that detail page. For example:

```
var detailPage = page.clickWorkItemTitle(page.workItemByTitle("Title Text 3"), "id3");
    expect(detailPage.getCreatedtime()).toBe('3 days ago');
    detailPage.clickWorkItemDetailCloseButton();
```
or
```
var detailPage = page.clickWorkItemTitle(page.workItemByIndex(3), "id3");
    expect(detailPage.getCreatedtime()).toBe('3 days ago');
    detailPage.clickWorkItemDetailCloseButton();
```

**Comment by naina-verma** [comment]
[test]

**Comment by naina-verma** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by naina-verma** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Pull request merged**